### PR TITLE
[Feature] Add project badges

### LINKED ISSUE #401: As a user I want to set up a customized badge for the project to better distinguish between projects
| Дано | Когда | Тогда |
| ----- | ------ | ------ | 
| У пользователя есть проект, который крутится в различных окружениях (prod, stage) | Он в настройка проекта в hawk | Может настроить бейджи |

### PR DESCRIPTION
**Pull request by ilyamore88**
Closes #401

### CODE REVIEW

**Review by nikmel2803 — APPROVED** [review]

File: src/components/project/ProjectHeader.vue [hunk 1]
```diff
@@ -14,7 +14,14 @@
         size="26"
```
Comment (neSpecc) [inline src/components/project/ProjectHeader.vue:14]: image

https://stage.hawk.so/project/5f8ddba049ab2f320eb76432

File: src/components/project/ProjectHeader.vue [hunk 2]
```diff
@@ -62,13 +96,31 @@ export default {
...
+    align-items: center;
     margin-left: 10px;
     color: var(--color-text-main);
     font-weight: bold;
     font-size: 15px;
+    overflow: hidden;
+  }
+
+  &__badge {
+    border-radius: 3px;
```
Comment (neSpecc) [inline src/components/project/ProjectHeader.vue:113]: do not repeat yourself, move it somewhere. For example `custom-properties.css`

File: src/components/aside/ProjectsMenuItem.vue [hunk 3]
```diff
@@ -119,13 +148,36 @@ export default {
       }
     }
 
-    &__name {
+    &__info {
+      overflow-x: hidden;
+    }
+
+    &__name-wrapper {
+      display: flex;
```
Comment (neSpecc) [inline src/components/aside/ProjectsMenuItem.vue:156]: i think, it's not needed. Just put the badge inside the `__name` element and make it `inline-block`

**Review by neSpecc — CHANGES_REQUESTED** [review]

**Review by neSpecc — APPROVED** [review]

**Review by neSpecc — CHANGES_REQUESTED** [review]
In project header: 
image

File: src/components/project/ProjectHeader.vue [hunk 4]
```diff
@@ -62,13 +97,33 @@ export default {
...
+    display: flex;
+    align-items: center;
     margin-left: 10px;
     color: var(--color-text-main);
     font-weight: bold;
     font-size: 15px;
+    overflow: hidden;
+  }
+
+  &__badge {
```
Comment (neSpecc) [inline src/components/project/ProjectHeader.vue:113]: duplicated code. Move it somewhere

**Review by neSpecc — CHANGES_REQUESTED** [review]

File: src/components/project/ProjectHeader.vue (hunk 2 above)
Comment (neSpecc) [reply src/components/project/ProjectHeader.vue]: actual

File: src/components/aside/ProjectsMenuItem.vue (hunk 3 above)
Comment (ilyamore88) [reply src/components/aside/ProjectsMenuItem.vue:156]: In name we use `v-html` prop. I don't know how to insert components inside v-html.

Comment (neSpecc) [reply src/components/aside/ProjectsMenuItem.vue:157]: > In name we use `v-html` prop. I don't know how to insert components inside v-html.

```vue
<div class="project-menu-item__name">
  <template v-html="name" /> 
  <project-badge />
</div>
```

File: src/components/aside/ProjectsMenuItem.vue [hunk 5]
```diff
@@ -72,7 +84,7 @@ export default {
       return 'No one catcher connected';
     },
     name() {
-      const escapedName = escape(this.project.name);
+      const escapedName = this.nameWithoutBadges(this.project.name);
```
Comment (neSpecc) [inline src/components/aside/ProjectsMenuItem.vue:87]: `escape` потерял

**Review by neSpecc — CHANGES_REQUESTED** [review]

**Review by neSpecc — APPROVED** [review]

**Pull request merged**